performanceMetrics/src/App/App.js

```diff
@@ -183,11 +184,14 @@ const App = (props) => {
 
 	useEffect(() => {
```

Comment:
You can merge with L179 use Effect. I don't think we should have 2 useEffect which have same dependecy.

Reply:
done